### DISCUSSION TITLE
Code of Conduct from `Community Standards`

### DISCUSSION THREAD

**Discussion by mvares** [FuelLabs/fuels-ts#2638] [Ideas]
Currently, we haven't completed all tasks from the `Community Standards`. To complete one of these, we need to add a `CODE_CONDUCT` file that meets our needs as an open-source project.

I was searching for some codes of conduct common in open-source projects, and I found this quite interesting:

https://www.contributor-covenant.org/

----
Relates to:
- #2129.

**Comment by petertonysmith94** [comment]
Maybe be worth just adding this to the issue:
- https://github.com/FuelLabs/fuels-ts/issues/2129

**Reply by mvares** [reply]
I didn’t see that I mentioned the wrong issue. Fixed it!

**Reply by petertonysmith94** [reply]
I was more suggesting at adding a comment to the issue, over creating a new discussion.

What is the discussion trying to achieve?

**Reply by mvares** [reply]
What the best option we can use.

I think we can close this discussion and conclude it in the issue.

**Reply by petertonysmith94** [reply]
I think that would be wise, keeps the context together 😃
